pycloudlib/cloud.py

```diff
@@ -76,7 +81,8 @@ def image_serial(self, image_id):
         """
         raise NotImplementedError
 
-    def get_instance(self, instance_id):
+    @abstractmethod
+    def get_instance(self, instance_id):  # () -> BaseInstance
```

Comment:
I don't know if we should be that flexible, but maybe we could have all the abstract methods to receive a `kwargs` parameter

Reply:
I'd default to not providing that now, just because adding them when we need to shouldn't require any additional pain and it keeps the calls a little cleaner and easier to understand if we never need them.